etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -173,6 +174,12 @@ DomainResponse handleResults(DomainRequest request) {
     }
 
     DomainResponse handleMetadata(DomainRequest request) {
+        if (Boolean.parseBoolean(request.getHeaders().get("load-test"))
+                && ApplicationContext.isPropertyPresent("REPORT_STREAM_URL_PREFIX")) {
```

Comment:
Is Application Context is set when we build the app?

Reply:
I'm having a hard time understanding your question here.  Can you rephrase it?

Reply:
No worries, I think I answered my own silly question. I forgot the that the whole point of env variables was to allow them to be set differently for each environment lol.